packages/cli/src/controller/init-controller.ts

```diff
@@ -237,13 +244,19 @@ export async function prepareProjectScaffold(projectPath: string): Promise<void>
   // remove all existing abis & handler files
   await prepareDirPath(path.join(projectPath, 'abis/'), false);
   await prepareDirPath(path.join(projectPath, 'src/mappings/'), true);
+  let manifest: Document.Parsed<ParsedNode, true> | string;
 
-  // clean datasource
-  const manifest = parseDocument(
-    (await fs.promises.readFile(path.join(projectPath, DEFAULT_MANIFEST), 'utf8')) as string
-  );
-  manifest.set('dataSources', new YAMLSeq());
-  await fs.promises.writeFile(path.join(projectPath, DEFAULT_MANIFEST), manifest.toString(), 'utf8');
+  if (fs.existsSync(path.join(projectPath, DEFAULT_MANIFEST))) {
```

Comment:
I think this needs to check for TS first, i know it shouldn't have a built yaml at this stage but you never know what might happen in the future